pele_platform/Utilities/Helpers/constraints/alpha_constraints.py

```diff
@@ -0,0 +1,228 @@
+from collections import defaultdict
+from prody.proteins.pdbfile import parsePDB
+import pele_platform.constants.constants as constants
+import pele_platform.Utilities.Helpers.map_atoms as map_atoms
+from PPP.checks_module import CheckforGaps
+
+
+terminal_constr_spring = 5
+backbone_constr_spring = 0.5
+
+CONSTR_ATOM = """{{ "type": "constrainAtomToPosition", "springConstant": {0}, "equilibriumDistance": 0.0, "constrainThisAtom": "{1}:{2}:{3}" }},"""
+CONSTR_DIST = """{{ "type": "constrainAtomsDistance", "springConstant": {}, "equilibriumDistance": {}, "constrainThisAtom": "{}:{}:{}", "toThisOtherAtom": "{}:{}:{}" }},"""
+CONSTR_CALPHA = """{{ "type": "constrainAtomToPosition", "springConstant": {2}, "equilibriumDistance": 0.0, "constrainThisAtom": "{0}:{1}:_CA_" }},"""
+
+
+class AlphaConstraints(object):
+    """
+    Class to parse alpha carbons in the PDB file and constrain them according to the user-defined settings.
+    """
+    def __init__(self, pdb, interval, backbone_spring, terminal_spring):
```

Comment:
I usually define the `__init__()` method to add a description for each of the input parameters it relies on.